lib/pure/math.nim

```diff
@@ -62,6 +62,9 @@ when defined(c) or defined(cpp):
   proc c_isnan(x: float): bool {.importc: "isnan", header: "<math.h>".}
     # a generic like `x: SomeFloat` might work too if this is implemented via a C macro.
 
+  proc c_copysign(x, y: float32): float32 {.importc: "copysignf", header: "math.h".}
```

Comment:
`"<math.>"`, not `"math.h"`; it matters for path resolution; we should actually (in future work) check that all stdlib imports use angle brackets `<>`